components/create_team/create_team.tsx

```diff
@@ -71,8 +76,18 @@ export default class CreateTeam extends React.PureComponent<Props & RouteCompone
             customDescriptionText,
             match,
             siteName,
+            isCloud,
+            isFreeTrial,
+            usageDeltas: {
+                teams: {
+                    active: usageDeltaTeams,
+                },
+            },
         } = this.props;
 
+        const teamsLimitReached = usageDeltaTeams >= 0;
+        const createTeamRestricted = isCloud && !isFreeTrial && teamsLimitReached;
```

Comment:
If I'm not mistaken, the limits are updated on free trial so we shouldn't need to check for free trial given that the limit will not be there

Reply:
Same as above comment.